### ISSUE TITLE
what is the name for this HISPID version?

### ISSUE THREAD

**Issue by AaronWilton** [hiscom/hispid#99]
my parting shot for the end of the week ;-)    maybe this has been agreed and I missed it???

What are we going to call this version? (current appears to have the name "HISPID Review 2015")
Will it be HISPID 6?

**Comment by nielsklazenga** [comment]
Yes, we discussed this. No more versions, just append the date to the identifier of the term in the history document.

**Comment by laurencepaine** [comment]
Interesting question. Given that we are now in the situation where it will perpetually be able to be revised (as per Niels previous note), is now the time to give it a unique moniker of HISPID2016 (or some other agreed name) to ensure there is never any confusion about this replacing the previous versions of hispid.

**Comment by nielsklazenga** [comment]
HISPID++

**Comment by AaronWilton** [comment]
Was looking at what some of the other TDWG standards do - some are using "Release X" notation 

but now time for another can of worms to add to this pile...  ;-)

HISPID = 'Herbarium Information Standards and Protocols for Interchange of Data'
so we now only have HISID... since we no longer do the protocols bit...

**Comment by nielsklazenga** [comment]
Ah, give it some time and we'll come up with a fitting backronym.

**Comment by nielsklazenga** [comment]
Easy, here you go:

# Herbarium Information Systems Produce Incomprehensible Data

**Comment by laurencepaine** [comment]
Couldn't we call it DIPSIH (HISPID backwards)...that's so close to DIPSHIT that I'm sure we could work wtih it.....

Sorry, been a long week. 

I think the Release X concept is something we really have to consider so that we hae an unambiguous referencing mechanism moving forward.

**Comment by AaronWilton** [comment]
personally I don't care if we stick with the HISPID acronym - give the nature of changes that we have made I would suggest that this would be a good juncture to mint something new....

**Comment by AaronWilton** [comment]
if we had to keep HISPID then how about something like - herbarium information standard profile for interchange of data  (but do we need to keep HISPID?)

**Comment by nielsklazenga** [comment]
I think we should keep the acronym HISPID (despite Laurence's very convincing argument) and leave it to people themselves what they want it to stand for. I don't think it is a good idea to look for another acronym.

**Comment by AaronWilton** [comment]
so....  HISPID, Release 6?

**Comment by laurencepaine** [comment]
I'm leaning towards HISPID Release 6. When we make the next major set of changes (and let's be really honest now, how many more concepts will we need to introduce to achieve our business of exchaning data?) it makes sense to me that we call it Release over Version.

I think we can live without needing to quantify what HISPID actually needs to stand for. It's just paying homage to the heritage of how we go to this point. 

+1 HISPID Release 6

**Comment by ben3000** [comment]
`echo preg_replace('/Release/', 'Version', 'HISPID Release 6');`

**Comment by afuchs1** [comment]
Hi all,

this makes sense to me....so +1

cheers Anne



Sent with Good (www.good.com)

________________________________

**Comment by nielsklazenga** [comment]
I am not keen on versioning HISPID at all other than using `issued` and `modified` dates (like Darwin Core does), but sure, whatever. I have to say though, if you are going to do versioning and use the consecutive numbering, this iteration of HISPID is a major version, not a release.
